crates/ruma-client-api/src/backup.rs

```diff
@@ -117,7 +117,10 @@ pub struct EncryptedSessionData {
     pub ciphertext: Base64,
 
     /// First 8 bytes of MAC key, encoded in base64.
-    pub mac: Base64,
+    pub mac: Option<Base64>,
```

Comment:
Hmm, previously I said that we'll want to have one or the other, but not both be an `Optional`.

But making the `mac` field optional is a breaking change and existing clients will reject keys where this field is missing.

I think we'll want to just add the new `mac2` field as an `Option` instead.